docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.install

```diff
@@ -35,6 +35,9 @@ function cgov_core_install() {
   // Install permissions for this module.
   _cgov_core_install_permissions($siteHelper);
   _cgov_core_remove_permissions($siteHelper);
+
+  // Set module weight so hooks run later than core's.
+  _cgov_core_set_module_weight();
```

Comment:
Changing the weight could have many unintended side effects. You can actually just change the ordering of hook implementations. https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/modules/custom/app_module/app_module.module#L210

Unless content moderation changes the order...